### PR TITLE
Add spiral-matrix exercise

### LINKED ISSUE #342: Implement Spiral Matrix exericse
Description can be found here:
https://github.com/exercism/problem-specifications/blob/master/exercises/spiral-matrix/description.md

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Closes #342

### CODE REVIEW

File: exercises/Exercises.sln [hunk 1]
```diff
@@ -219,6 +219,7 @@ EndProject
 Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "RotationalCipher", "rotational-cipher\RotationalCipher.fsproj", "{91D0A5E0-E39C-472B-87FB-6DD17336AA22}"
 EndProject
 Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "IsbnVerifier", "isbn-verifier\IsbnVerifier.fsproj", "{531D1AB8-1297-4DE2-9B9B-3742A648EDB7}"
```
Comment (robkeim) [inline exercises/Exercises.sln]: @ErikSchierboom is there a reason that `IsbnVerifier` no longer has an `EndProject`?

**Comment by robkeim** [comment]
Changes look good to me other than the weird change in `Exercises.sln`.

File: exercises/Exercises.sln (hunk 1 above)
Comment (ErikSchierboom) [reply exercises/Exercises.sln]: Whoops, merge error! I've fixed it.

**Comment by robkeim** [comment]
Thanks @ErikSchierboom! 🎉

**Pull request merged**